### DISCUSSION TITLE
v24.12.22

### DISCUSSION THREAD

**Discussion by davidrapan** [davidrapan/ha-solarman#340] [Announcements]
## What's Changed
* [refactor: Move pysolarmanv5 enhancements to submodule](https://github.com/davidrapan/ha-solarman/commit/2ba7ea041285d7c6cd6fbc157f473901d673064c)
* [refactor: Rename write register methods in conformity w/ umodbus](https://github.com/davidrapan/ha-solarman/commit/8381431550dfa52af2a60e37d4dbc64bb41bfe08)
* Fix version info in Device sensor attributes

**Full Changelog**: https://github.com/davidrapan/ha-solarman/compare/v24.12.20...v24.12.22

<hr /><em>This discussion was created from the release <a href='https://github.com/davidrapan/ha-solarman/releases/tag/v24.12.22'>v24.12.22</a>.</em>

**Comment by sofkaski** [comment]
I installed `v24.12.22` and based on my silk glove testing it is working.
